src/actions/__tests__/verify-action-store-consistency.test.js

```diff
@@ -122,3 +127,18 @@ describe('make sure action creators leave store in a consitent state', () => {
       .not.toThrow()
   })
 })
+
+describe('setKernelState', () => {
+  it('createValidatedReducer should throw an error if we pass an invalid arg into setKernelState', () => {
+    expect(() => store.dispatch(actions.setKernelState('fake state')))
+      .toThrowError(SchemaValidationError)
+    expect(() => store.dispatch(actions.setKernelState(12342323)))
+      .toThrowError(SchemaValidationError)
+  })
+  it('passes on correct set of enums for setKernelState', () => {
+    const enums = ['KERNEL_LOADING', 'KERNEL_LOAD_ERROR', 'KERNEL_ERROR', 'KERNEL_IDLE', 'KERNEL_BUSY']
```

Comment:
i think you should be able to import the enum from the state-schema directly, so that this test will always be in sync with the possible option in case we need to add/remove anything later on